src/main/java/Storage.java

```diff
@@ -0,0 +1,111 @@
+import java.io.*;
```

Comment:
I will be better if you listed your imports.

Reply:
Based on our course's coding style, wildcard imports are not recommended.